docs/nddata/index.rst

```diff
@@ -328,35 +328,43 @@ the mask, metadata, etc are discarded:
 
 .. doctest-requires:: skimage
 
+    >>> from astropy.utils import minversion
+    >>> from astropy.utils.compat import NUMPY_LT_1_16
     >>> from astropy.nddata import block_reduce, block_replicate
-    >>> smaller = block_reduce(ccd, 4)
-    >>> smaller
-    array(...)
-    >>> plt.imshow(smaller, origin='lower')  # doctest: +SKIP
+    >>> SKIMAGE_LT_1_14_2 = not minversion('skimage', '0.14.2')
+    >>> # Version check is necessary here due to upstream incompatibility.
+    >>> if not SKIMAGE_LT_1_14_2 or (SKIMAGE_LT_1_14_2 and NUMPY_LT_1_16):
```

Comment:
I'm not convinced that hacking an example on the index docs page for a subpackage is the best way.
How difficult would it be to also specify a minimum version number for `doctest-requires`, cc @drdavella?

Reply:
We can also wait for `skimage` 0.14.2 to be released and see if that automagically fixes things, as some others suggested.